inc/options/minify.php

```diff
@@ -1,78 +1,156 @@
 <?php
 namespace W3TC;
 
-if ( !defined( 'W3TC' ) )
+if ( ! defined( 'W3TC' ) ) {
 	die();
-
+}
 ?>
-<?php include W3TC_INC_DIR . '/options/common/header.php'; ?>
+<?php require W3TC_INC_DIR . '/options/common/header.php'; ?>
 
 <script type="text/javascript">/*<![CDATA[*/
 	var minify_templates = {};
-	<?php foreach ( $templates as $theme_key => $theme_templates ): ?>
-	minify_templates['<?php echo addslashes( $theme_key ); ?>'] = {};
-	<?php foreach ( $theme_templates as $theme_template_key => $theme_template_name ): ?>
-	minify_templates['<?php echo addslashes( $theme_key ); ?>']['<?php echo addslashes( $theme_template_key ); ?>'] = '<?php echo addslashes( $theme_template_name ); ?>';
-	<?php endforeach; ?>
+	<?php foreach ( $templates as $theme_key => $theme_templates ) : ?>
+		minify_templates['<?php echo esc_html( addslashes( $theme_key ) ); ?>'] = {};
+		<?php foreach ( $theme_templates as $theme_template_key => $theme_template_name ) : ?>
+			minify_templates['<?php echo esc_html( addslashes( $theme_key ) ); ?>']['<?php echo esc_html( addslashes( $theme_template_key ) ); ?>'] = '<?php echo esc_html( addslashes( $theme_template_name ) ); ?>';
+		<?php endforeach; ?>
 	<?php endforeach; ?>
 /*]]>*/</script>
 
-<form action="admin.php?page=<?php echo $this->_page; ?>" method="post">
+<form action="admin.php?page=<?php echo esc_attr( $this->_page ); ?>" method="post">
 	<p>
-		<?php echo sprintf( __( 'Minify via %s is currently %s.', 'w3-total-cache' ), Cache::engine_name( $this->_config->get_string( 'minify.engine' ) ) , '<span class="w3tc-' . ( $minify_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>' ); ?>
+		<?php
+		echo wp_kses(
+			sprintf(
+				// translators: 1 minify engine name, HTML span tag indicating engine enabled/disabled.
+				__(
+					'Minify via %1$s is currently %2$s.',
+					'w3-total-cache'
+				),
+				Cache::engine_name( $this->_config->get_string( 'minify.engine' ) ),
+				'<span class="w3tc-' . ( $minify_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>'
```

Comment:
Use `esc_html__()`.

Reply:
Switched to esc_html__ for both instances